xbmc/utils/log.cpp

```diff
@@ -282,7 +282,7 @@ void CLog::FormatAndLogInternal(spdlog::level::level_enum level,
   auto message = fmt::vformat(format, args);
 
   // fixup newline alignment, number of spaces should equal prefix length
-  FormatLineBreaks(message);
+  StringUtils::Replace(message, "\n", "\n                                                   ");
```

Comment:
Can anybody tell me what "prefix length" is and why we need this fixup?

Reply:
It's purely cosmetical, right? To make multi-line log message more readable.

But this does not really wor, because the name of the logger is not always "general". It can vary in length. Clients can define whatever logger name they want to use. => https://github.com/xbmc/xbmc/blob/master/xbmc/utils/log.h#L87 - this method gets called from several places with freely chosen logger names.